ex2/Date time.py

```diff
@@ -0,0 +1,56 @@
+from bs4 import BeautifulSoup
+from urllib.request import urlopen
+import re
+
+
+class Datetime:
+    _html_page = BeautifulSoup(urlopen("https://www.timeanddate.com/worldclock/israel/jerusalem").read(), "html.parser")
```

Comment:
This line is executed once, when the class is created. So different calls to now() give yhe same time. This is wrong.
It should be a local variable as it was before.

You can parse zone time and assign it to the object - make zonetime() a method instead of a staticmethod.